pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file defining the project's dependencies:
+
+`composer init`
+
+- Create a new project from a given version of an existing package:
```

Comment:
I am having difficulty grasping this line. What does a "version" of an existing package have to do with creating a new project. I mean I can create a new project without any package dependencies at all, right ? And what does it mean when you say create a project "from" a package ?